presto-generation-processor/README.md

```diff
@@ -0,0 +1,16 @@
+# Presto Code Generation Annotation Prococessors
+
+## Links
+
+### Deor's 3 part series about code generation via annotations
+This is linked from everywhere under the sun as a reference, to the point that it seems to be the de-facto official reference on generating code via annotation processing.
+The suggestion to use Velocity and the project structure both come from these blog posts.
+- [Part 1](https://deors.wordpress.com/2011/09/26/annotation-types/)
+- [Part 2](https://deors.wordpress.com/2011/10/08/annotation-processors/)
+- [Part 3](https://deors.wordpress.com/2011/10/31/annotation-generators/)
+
+### Maven is broken by design
```

Comment:
Remove this title and links. It's a mostly outdated, inflammatory rant. While he's not completely wrong and there may be some useful bits, that's outweighed by the inflammatory nature and is not something we want to promote.

If there are things you want to document, do that here directly in a neutral POV.